lib/edoc/src/edoc.erl

```diff
@@ -55,11 +55,17 @@
 	 read_comments/1, read_comments/2,
 	 read_source/1, read_source/2]).
 
+-export_type([module/0]).
```

Comment:
Can not find a type named <code>module</code>. Should it be <code>edoc_module</code>?